docs/firefighting/index.md

```diff
@@ -890,6 +890,11 @@ Resources:
 
 Formplayer sometimes fails on deploy due to a startup task (which will hopefully be resolved soon).  The process may not fail, but formplayer will still return failure responses. You can try just restarting the process with `sudo supervisorctl restart all` (or specify the name if it's a monolithic environment)
 
+A formplayer machine may need to be restarted for a number of reasons in which case you can run:
+```
+cchq <env> service formplayer restart --limit=formplayer_bXXX
```

Comment:
Can you do multiple machines with `--limit`?

Reply:
Looking at this https://github.com/dimagi/commcare-cloud/blob/43ad450e36993a12d5af602cfcf20f2ca0347cbf/src/commcare_cloud/commands/ansible/service.py#L84 and the rest of the file I'm 99% sure the answer is yes. I also ran the `status` command instead of `restart` with multiple machines so definite yes. I will update the documentation to reflect that.

Reply:
Beautiful, thank you.